src/rospack_backcompat.cpp

```diff
@@ -40,7 +40,8 @@ namespace rospack
 int
 ROSPack::run(int argc, char** argv)
 {
-  rospack::Rospack rp;
+  // allow caching of results between calls (of same process)
+  static rospack::Rospack rp;
```

Comment:
@dirk-thomas Did you already had a chance to discuss how to proceed on the rospack caching?